.babelrc.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
 	env: {
 		build: {
 			ignore: ['**/*.test.ts', '**/__mocks__/**'],
-			presets: [['@babel/env', { targets: { node: '14.19.0' } }], '@babel/typescript'],
```

Comment:
When targeting, we'll want to match whatever the lowest end of the range would be.  If we change the range then that would match that (i.e. `16.0.0`).  If we leave this higher range then it would be `16.19.0`).